src/legacy/components/AddressInputPanel/index.tsx

```diff
@@ -132,6 +132,7 @@ export function AddressInputPanel({
               pattern="^(0x[a-fA-F0-9]{40})$"
               onChange={handleInput}
               value={value}
+              onFocus={(e) => e.target.select()}
```

Comment:
is it worth it to make an utility function so we just add the `autofocus` where needed?